openapi/sinch.sms/openapi.yaml

```diff
@@ -0,0 +1,1657 @@
+openapi: 3.0.0
+info:
+  title: API Overview
+  version: "v1"
+  description: 
+    This is a generated connector for [Sinch SMS API v1](https://www.sinch.com/) OpenAPI specification. 
+
+    Sinch SMS API is the one of the easiest APIs we offer and enables you to add fast and reliable global SMS to your applications. It allows for sending single messages, scheduled batch sending, message templates and "Only list messages sent from this sender number. Multiple originating numbers can be comma separated.
+  x-ballerina-init-description:
+    The connector initialization requires setting the API credentials.
+
+    Create a [Sinch account](https://www.sinch.com/) and obtain tokens by following [this guide](https://developers.sinch.com/docs/sms/api-reference/#authentication).
+  x-ballerina-display:
+    label: sinch 
```

Comment:
Don't we need to give it as Sinch SMS? Image name is it as `sinch.sms`.

Reply:
Yes, `Sinch SMS` will be the correct name